apps/spruce/src/pages/waterfall/InactiveVersionsButton/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+import Button from "@leafygreen-ui/button";
+import { palette } from "@leafygreen-ui/palette";
+import pluralize from "pluralize";
+import { DisplayModal } from "components/DisplayModal";
+import Icon from "components/Icon";
+import { size } from "constants/tokens";
+import { WaterfallQuery } from "gql/generated/types";
+import { VersionLabel } from "../VersionLabel";
+
+const { gray } = palette;
+interface Props {
+  versions: WaterfallQuery["waterfall"]["versions"][0]["inactiveVersions"];
+  containerHeight: number | undefined;
+}
+export const InactiveVersionsButton: React.FC<Props> = ({
+  containerHeight,
+  versions,
+}) => {
+  const [modalOpen, setModalOpen] = useState(false);
+  return (
+    <>
+      <DisplayModal
+        data-cy="inactive-versions-modal"
+        open={modalOpen}
+        setOpen={setModalOpen}
+        title={`${versions?.length} Inactive ${pluralize("Version", versions?.length)}`}
+      >
+        {versions?.map((version) => (
+          <StyledVersionLabel
```

Comment:
Cool to reuse the VersionLabel component 😎 for this instance, I think we should remove the text trim present in that component in case of long commits. The `title` attribute is also redundant.

We could also opt to use our regular font size instead of the small version, whatever you think.

Reply:
I removed the title property when the message is full length

Reply:
also made a size prop